starters/deno-oak-denodb/.env.example

```diff
@@ -0,0 +1,11 @@
+API_HOST_URL='http://localhost:3333'
```

Comment:
It seems that the port information is duplicated in these two lines. What do you think about having something like `API_HOST='http://localhost'` instead? And then you concatanate the two env vars where needed.